.uberenv_config.json

```diff
@@ -4,7 +4,7 @@
 "package_final_phase": "initconfig",
 "package_source_dir": "../..",
 "spack_url": "https://github.com/spack/spack.git",
-"spack_commit": "bb1cd430c0d6f8bebfb655c65244a25bcc325d8d",
+"spack_commit": "ddc6e233c70273dd5dfd8dab31729eaf538693b8",
```

Comment:
Is there a reason that we don't pin to a Spack release?

Reply:
We are constantly needing to move forward due to specific fixes.